examples/persistent_homology_graphs.ipynb

```diff
@@ -0,0 +1,659 @@
+{
```

Comment:
<p>I wonder if we can outsource this content to the VR tutorial? Otherwise we will end up always having to explain persistent homology in each notebook :D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/462/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I'll think about this!</p><p> </p>

Reply:
<p>Migrated to the VR notebook now, with an improved explanation. Cross-referenced VR notebook in this one for this explanation.</p>